packages/@react-aria/button/src/useButton.ts

```diff
@@ -87,8 +97,13 @@ export function useButton(props: AriaButtonProps, ref: RefObject<HTMLElement>):
   return {
     isPressed, // Used to indicate press state for visual
     buttonProps: mergeProps(interactions, {
+      id,
```

Comment:
To discuss: I found it annoying when writing the examples in the aria docs that some dom props weren't passed through. Perhaps we should move filterDOMProps into aria instead of spectrum?

Reply:
I think that makes sense? it seems more aria related anyways